full-service/src/test_utils.rs

```diff
@@ -365,12 +333,14 @@ pub fn add_block_with_db_txos(
     let outputs: Vec<TxOut> = output_txo_ids
         .iter()
         .map(|txo_id_hex| {
-            mc_util_serial::decode(
-                &Txo::get(&txo_id_hex.to_string(), &wallet_db.get_conn().unwrap())
-                    .unwrap()
-                    .txo,
-            )
-            .unwrap()
+            let txo = &Txo::get(&txo_id_hex.to_string(), &wallet_db.get_conn().unwrap()).unwrap();
+            ledger_db
+                .get_tx_out_by_index(
+                    ledger_db
+                        .get_tx_out_index_by_public_key(&txo.public_key().unwrap())
+                        .unwrap(),
+                )
+                .unwrap()
         })
         .collect();
 
```

Comment:
This function may not be needed, or even work anymore? Since the txo doesn't exist in the ledger yet, the call to get_tx_out_by_public_key will just fail